react/features/deep-linking/components/DeepLinkingDesktopPage.web.tsx

```diff
@@ -134,6 +143,18 @@ const DeepLinkingDesktopPage: React.FC<WithTranslation> = ({ t }) => {
                             : t(`${_TNS}.descriptionWithoutWeb`, { app: desktop?.appName })
                     }
                 </div>
+                <div className = { styles.descriptionLabel }>
+                    {
+                        t(`${_TNS}.noMobileApp`)
+                    }
+                </div>
+                <div className = { styles.descriptionLabel }>
+                    <a href = { generateDownloadURL() }>
```

Comment:
Could we please put the download link on the same like as "You don't have the app?"